config/exports/exports.yml

```diff
@@ -188,3 +188,7 @@ hamilton_lrc_pbmc_defrost_pbs_to_lrc_pbmc_pools:
     - wells.qc_results
     - wells.transfer_requests_as_target.source_asset
   ancestor_purpose: LRC PBMC Defrost PBS
+hamilton_lrc_ht_5p_cherrypick_to_lrc_ht_5p_gex_dil:
```

Comment:
The dil plate is the first plate in the GEX library prep part of the pipeline. i.e. it will branch in future at the parent cherrypick plate into GEX, TCR and BCR library preps. Up till the cherrypick it is 5p for scRNA, then we get more specific for the library preps. For phase 1 we just have GEX library prep. This purpose naming is the same in the Bespoke pipeline where GEX is also included in the purpose names.
So I think it is right.